### PR TITLE
Compose per integration metadata.yaml and ops_agent_metadata.yaml files based on the content in ops-agent repo.

### PR DESCRIPTION
**Pull request by stackdriver-instrumentation-release**

### CODE REVIEW

**Comment by sophieyfang** [comment]
DO_NOT_MERGE until Varnish and Jetty public doc are published. @EvanSimpson i think you are subscribed to Mary's doc CLs. So once it's published, then you can merge this PR in.

As for the description change, that was discussed in the CL comment if you remember, the metadata.yaml file now has a new field named as `app_url` that stored the information. Let me know if it's something that FSM metadata would like to have :)

**Comment by sophieyfang** [comment]
Varnish and Jetty public doc have been released. @EvanSimpson feel free to merge this PR

File: integrations/elasticsearch/metadata.yaml [hunk 1]
```diff
@@ -3,12 +3,9 @@ short_name: Elasticsearch
 display_name: Elasticsearch
 description: 'Elasticsearch is an open-source search server, based on the
 
-  [Lucene](https://lucene.apache.org/) search library. It runs in a Java virtual
+  [Lucene](https://lucene.apache.org/){: .external} search library. It runs in a Java
```
Comment (EvanSimpson) [inline integrations/elasticsearch/metadata.yaml:6]: @sophieyfang what are these `{: .external}` annotations? This doesn't look like standard markdown.

Comment (sophieyfang) [reply integrations/elasticsearch/metadata.yaml:6]: These are the ones required by Mary Tabasko for the google public page.

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**